packages/venia-ui/lib/components/CartPage/GiftCards/giftCards.js

```diff
@@ -227,12 +226,16 @@ const GiftCards = props => {
 
     return (
         <div className={classes.root}>
-            <div className={classes.entryForm}>
-                <Form getApi={setFormApi}>{cardEntryContents}</Form>
-            </div>
+            <div className={classes.entryForm}>{cardEntryContents}</div>
             {appliedGiftCards}
         </div>
     );
 };
 
-export default GiftCards;
+export default props => {
+    return (
+        <Form>
+            <GiftCards {...props} />
+        </Form>
+    );
+};
```

Comment:
Do we use this pattern anywhere else?

Can we incorporate the `Form` into the return of `GiftCards` for consistency?

Reply:
Nevermind, I see now that you needed this to use the `useFormApi` hook in the talon, right?

Reply:
Yeah, the `getApi` pattern that is presented in this code before this PR is causing `illegal setState` errors after the react upgrade. Also, it is a pattern we don't use often. So I changed it to what we usually do, get `formApi` using `useFormApi` in the talon.